.github/workflows/tests.yml

```diff
@@ -0,0 +1,22 @@
+name: Run tests
```

Comment:
We can easily discard this test by configuring the local developmental IDE to auto-format code with Black.

Reply:
I have configured my VS Code. Will remove this.